### PR TITLE
[pt] Add pt/content/docs/concepts/observability-primer.md

### PR DESCRIPTION
**Pull request by EzzioMoreira**
Add a localization to the observability-primer.md.

Issue: https://github.com/open-telemetry/opentelemetry.io/issues/4922

### CODE REVIEW

**Review by emdneto — APPROVED** [review]

File: content/pt/docs/concepts/observability-primer.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+para sistemas distribuídos, que costumam ter problemas não determinísticos ou
+que são muito complicados de reproduzir localmente.
+
+Para entender o rastreamento distribuído, você precisa compreender a função de
+cada um de seus componentes: logs, trechos e rastros.
+
+### Logs
+
+Um **log** é uma mensagem com registro de data e hora emitida por serviços ou
+outros componentes. Ao contrário dos [rastros](#rastros-distribuídos), eles não
```
Comment (svrnm) [inline content/pt/docs/concepts/observability-primer.md:72]:
```suggestion
outros componentes. Ao contrário dos [rastros](#rastros-distribuídos), eles não
```
Do not translate hashes, instead add a name to the line (see comment below)

File: content/pt/docs/concepts/observability-primer.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+| `http.route`                | `"/webshop/articles/:article_id"`                                                  |
+| `http.response.status_code` | `200`                                                                              |
+| `client.address`            | `"192.0.2.4"`                                                                      |
+| `client.socket.address`     | `"192.0.2.5"` (o cliente passa por um proxy)                                       |
+| `user_agent.original`       | `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:72.0) Gecko/20100101 Firefox/72.0"` |
+
+Para mais informações sobre trechos e como eles se relacionam com o
+OpenTelemetry, consulte [Trecho](/docs/concepts/signals/traces/#spans).
+
+### Rastros distribuídos
```
Comment (svrnm) [inline content/pt/docs/concepts/observability-primer.md:128]:
```suggestion
### Rastros distribuídos {#the-untranslated-english-title}
```

Do this to keep the anchor

**Review by svrnm — CHANGES_REQUESTED** [review]

**Comment by svrnm** [comment]
```
pt/docs/index.html
  hash does not exist --- pt/docs/index.html --> concepts/observability-primer/#what-is-observability
pt/docs/concepts/observability-primer/index.html
  hash does not exist --- pt/docs/concepts/observability-primer/index.html --> #distributed-traces
  hash does not exist --- pt/docs/concepts/observability-primer/index.html --> #spans
========================================================================
```

there are a few more hashes that require some of that treatment :)

File: content/pt/docs/concepts/observability-primer.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+| `http.route`                | `"/webshop/articles/:article_id"`                                                  |
+| `http.response.status_code` | `200`                                                                              |
+| `client.address`            | `"192.0.2.4"`                                                                      |
+| `client.socket.address`     | `"192.0.2.5"` (o cliente passa por um proxy)                                       |
+| `user_agent.original`       | `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:72.0) Gecko/20100101 Firefox/72.0"` |
+
+Para mais informações sobre trechos e como eles se relacionam com o
+OpenTelemetry, consulte [Trecho](/docs/concepts/signals/traces/#spans).
+
+### Rastros distribuídos {#the-untranslated-english-title}
```
Comment (lgfa29) [inline content/pt/docs/concepts/observability-primer.md:128]:
```suggestion
### Rastros distribuídos
```

File: content/pt/docs/concepts/observability-primer.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+```text
+I, [2021-02-23T13:26:23.505892 #22473]  INFO -- : [6459ffe1-ea53-4044-aaa3-bf902868f730] Started GET "/" for ::1 at 2021-02-23 13:26:23 -0800
+```
+
+Logs não são suficientes para rastrear a execução do código, pois geralmente não
+apresentam informações contextuais, como de onde foram chamados.
+
+Eles se tornam muito mais úteis quando incluídos como parte de um
+[trecho](#spans) ou quando são correlacionados com um rastro e um trecho.
```
Comment (lgfa29) [inline content/pt/docs/concepts/observability-primer.md:88]:
```suggestion
[trecho](#trechos) ou quando são correlacionados com um rastro e um trecho.
```

File: content/pt/docs/concepts/observability-primer.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+para sistemas distribuídos, que costumam ter problemas não determinísticos ou
+que são muito complicados de reproduzir localmente.
+
+Para entender o rastreamento distribuído, você precisa compreender a função de
+cada um de seus componentes: logs, trechos e rastros.
+
+### Logs
+
+Um **log** é uma mensagem com registro de data e hora emitida por serviços ou
+outros componentes. Ao contrário dos [rastros](#distributed-traces), eles não
```
Comment (lgfa29) [inline content/pt/docs/concepts/observability-primer.md:72]:
```suggestion
outros componentes. Ao contrário dos [rastros](#rastros-distribuídos), eles não
```

**Review by lgfa29 — COMMENTED** [review]
Não sei o que aconteceu, mas esses valores acabaram mudando em comparação com https://github.com/open-telemetry/opentelemetry.io/pull/4946

File: content/pt/docs/concepts/observability-primer.md (hunk 3 above)
Comment (svrnm) [reply content/pt/docs/concepts/observability-primer.md:128]:
```suggestion
### Rastros distribuídos {#distributed-traces}
```

as of today we keep the anchors untranslated.

File: content/pt/docs/concepts/observability-primer.md (hunk 4 above)
Comment (svrnm) [reply content/pt/docs/concepts/observability-primer.md:88]: as of today we keep the anchors untranslated, so the Spans section should have

```
# Trechos {#spans}
```

as title

Comment (lgfa29) [reply content/pt/docs/concepts/observability-primer.md:88]: Ohh TIL, thanks!

Should we update all pages that have already been translated so every title matches the original anchor?

Comment (EzzioMoreira) [reply content/pt/docs/concepts/observability-primer.md:88]: @lgfa29 good question, in this example: 
Would it look like this?
pt/docs/concepts/observability-primer.md
`## O que é Observabilidade? {#what-is-observability}`

**Review by chalin — APPROVED** [review]

**Comment by chalin** [comment]
All checks are green, and @svrnm's concerns have been addressed (and is OOO), so I'm approving and merging on behalf of @open-telemetry/docs-maintainers & @svrnm.

**Pull request merged**